src/Components/PostList.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from './Post';
+
+const PostList = ({ posts }) => (
+  <div className="post-list">
+    {posts.map(post => <Post post={post} key={post.id} />)}
```

Comment:
Четыре слова post подряд - не очень гуд
```suggestion
    {posts.map(postData => <Post post={postData} key={post.id} />)}
```